__tests__/app.test.js

```diff
@@ -496,4 +496,37 @@ describe("GET /api/reviews/:review_id (comment count)",() => {
             expect(review.review_id).toEqual(2)
         })
     })
+})
+
+// DELETE /api/comments/:comment_id
+describe("DELETE /api/comments/:comment_id",() => {
+    test("if given valid comment_id, status 204, return nothing - check that number of comments is reduced",() => {
```

Comment:
This test could be simplified slightly as we're using test data so know that the comment_id exits.

I like the check afterwards to verify that the comment has been deleted!